app/bundles/PageBundle/EventListener/BuilderSubscriber.php

```diff
@@ -250,7 +269,15 @@ public function onPageBuild(Events\PageBuilderEvent $event)
                 'video-camera',
                 'MauticCoreBundle:Slots:gatedvideo.html.php',
                 GatedVideoType::class,
-                600
+                300
+            );
+            $event->addSlotType(
+                'dwc',
+                'Dynamic Content',
```

Comment:
We should probably be translating all these slot names. Didn't realize we weren't.